### ISSUE TITLE
Feature: Allow client to be created out of an existing browser ws

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#481]
This feature will allow puppeteering of remote sessions.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.46](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.45...1.9.46) (2020-05-24)

- Release 1.9.46 [`b359df9`](https://github.com/open-wa/wa-automate-nodejs/commit/b359df974f756d741acf0137d439e7186e59e728)
- EXPERIMENTAL FEATURE: implemented connect via `browserWSEndpoint` #481 [`9942a76`](https://github.com/open-wa/wa-automate-nodejs/commit/9942a76778e9de1bc64484ee73299c0a5c62baff)

**Comment by smashah** [comment]
https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#browserwsendpoint
